semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -16,9 +16,139 @@
 
 // DO NOT EDIT, this is an Auto-generated file from
 // buildscripts/semantic-convention/templates/SemanticAttributes.java.j2
+@SuppressWarnings("unused")
 public final class SemanticAttributes {
   /** The URL of the OpenTelemetry schema for these keys and values. */
-  public static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.13.0";
+  public static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.15.0";
+
+  /**
+   * The type of the exception (its fully-qualified class name, if applicable). The dynamic type of
+   * the exception should be preferred over the static type in languages that support it.
+   */
+  public static final AttributeKey<String> EXCEPTION_TYPE = stringKey("exception.type");
+
+  /** The exception message. */
+  public static final AttributeKey<String> EXCEPTION_MESSAGE = stringKey("exception.message");
+
+  /**
+   * A stacktrace as a string in the natural representation for the language runtime. The
+   * representation is to be determined and documented by each language SIG.
+   */
+  public static final AttributeKey<String> EXCEPTION_STACKTRACE = stringKey("exception.stacktrace");
+
+  /** The name identifies the event. */
+  public static final AttributeKey<String> EVENT_NAME = stringKey("event.name");
+
+  /**
+   * The domain identifies the context in which an event happened. An event name is unique only
+   * within a domain.
+   *
+   * <p>Notes:
+   *
+   * <ul>
+   *   <li>An {@code event.name} is supposed to be unique only in the context of an {@code
+   *       event.domain}, so this allows for two events in different domains to have same {@code
+   *       event.name}, yet be unrelated events.
+   * </ul>
+   */
+  public static final AttributeKey<String> EVENT_DOMAIN = stringKey("event.domain");
+
+  /**
+   * The Amazon Resource Name (ARN) of an <a
+   * href="https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_instances.html">ECS
+   * container instance</a>.
+   */
+  public static final AttributeKey<String> AWS_ECS_CONTAINER_ARN =
+      stringKey("aws.ecs.container.arn");
```

Comment:
Something's still not right with the script. These are defined in the [ecs resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/94c9c75c4f82fbda08bddff02ce9a0c582fdd55c/specification/resource/semantic_conventions/cloud_provider/aws/ecs.md).